lib/child_process.js

```diff
@@ -50,7 +50,24 @@ exports.fork = function(modulePath /*, args, options*/) {
 
   args = execArgv.concat([modulePath], args);
 
-  if (!Array.isArray(options.stdio)) {
+  if (typeof options.stdio == 'string') {
+    const stdioStringToArray = (option) => {
```

Comment:
I think this should be moved outside of `fork()` since it does not rely on any variables in the current scope, where it should use the `function stdioStringToArray(option) { ... }` format.